step-certificates/templates/secrets.yaml

```diff
@@ -40,12 +51,20 @@ metadata:
   name: {{ include "step-certificates.fullname" . }}-secrets
   namespace: {{ .Release.Namespace }}
 stringData:
+  {{- if .Values.inject.secrets.certificate_issuer.enabled }}
+  certificate_issuer_key: |-
+    {{- .Values.inject.secrets.certificate_issuer.key | nindent 4 }}
+  {{- end }}
```

Comment:
Only if the key is given, for JWK is optional.

Reply:
fixed.